code/go/0chain.net/miner/protocol_block.go

```diff
@@ -354,7 +355,12 @@ func (mc *Chain) VerifyBlock(ctx context.Context, b *block.Block) (
 		if err != nil {
 			return nil, err
 		}
-		cost += c //810
+
+		newCost, err := maths.SafeAddInt(cost, c)
```

Comment:
The `cost` is small number, not tokens, it represents the general CPU, RAM, etc resource cost for executing a smart contract. So we can ignore the overflow checking for them.